messages/core.md

```diff
@@ -34,6 +34,10 @@ Errors encountered:
 
 Error authenticating with the refresh token due to: %s
 
+# invalidSfdxAuthUrlError
+
+Invalid SFDX auth URL. Must be in the format "force://<clientId>:<clientSecret>:<refreshToken>@<instanceUrl>". Note that the "instanceUrl" inside the SFDX auth URL doesn\'t include a protocol ("https://").
```

Comment:
```suggestion
Invalid SFDX authorization URL. Must be in the format "force://<clientId>:<clientSecret>:<refreshToken>@<instanceUrl>". Note that the "instanceUrl" inside the SFDX authorization URL doesn\'t include the protocol ("https://"). Run "org display --target-org" on an org to see an example of an SFDX authorization URL. 
```

Reply:
Okay?  I updated the name of the URL to match what's in the "org login sfdx-url" help.  And I think it's okay not to put that whole stuff about commands always using HTTPS here.  One can assume at this point that they've run --help on "org login sfdx-url".